pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java

```diff
@@ -130,9 +130,11 @@ private static void generateBrokerBasicMetrics(PulsarService pulsar, SimpleTextO
         parseMetricsToPrometheusMetrics(new ManagedLedgerMetrics(pulsar).generate(),
                 clusterName, Collector.Type.GAUGE, stream);
 
-        // generate managedCursor metrics
-        parseMetricsToPrometheusMetrics(new ManagedCursorMetrics(pulsar).generate(),
-                clusterName, Collector.Type.GAUGE, stream);
+        if(includeManagedCursorMetrics){
```

Comment:
Why not use `pulsar.getConfiguration().isExposeManagedCursorMetricsInPrometheus()
` here? please note the code style `if (...) {}`.

Reply:
Just for being consistent with `includeConsumerMetrics`, `includeProducerMetrics` etc.